--- ding-1.7/ding.desktop~	2009-04-15 08:25:28.000000000 +0200
+++ ding-1.7/ding.desktop	2010-09-14 13:16:11.815055141 +0200
@@ -2,10 +2,10 @@
 Encoding=UTF-8
 Name=Ding: Dictionary English-German
 Name[de]=Ding: Wörterbuch Englisch-Deutsch
+Name[pl]=Ding: Słownik Angielsko-Niemiecki
 GenericName=ding
-GenericName[de]=ding
-Exec=/usr/bin/ding
-Icon=ding.png
+Exec=ding
+Icon=ding
 Terminal=0
 Type=Application
-Categories=Application;Utility;X-Red-Hat-Extra;
+Categories=Office;Dictionary;
